client/packages/common/src/ui/layout/tables/material-react-table/usePaginatedMaterialTable.tsx

```diff
@@ -131,9 +133,5 @@ export const usePaginatedMaterialTable = <T extends MRT_RowData>({
     [rowSelection]
   );
 
-  const resetRowSelection = () => {
-    table.resetRowSelection();
-  };
```

Comment:
I think its fine to use table.resetRowSelection directly, rather than a separate fn definition like this